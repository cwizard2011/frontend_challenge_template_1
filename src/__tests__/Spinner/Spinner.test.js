import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Spinner from '../../Spinner';

describe('<Spinner />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Spinner />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
