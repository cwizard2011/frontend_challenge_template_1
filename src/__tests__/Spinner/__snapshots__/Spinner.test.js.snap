// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> renders without crashing 1`] = `
<div
  className="spinner"
/>
`;
